plugins/rich-editor/README.md

```diff
@@ -0,0 +1,218 @@
+# Rich Editor (WIP)
+
+A highly-functional WYSIWYG editor for Vanilla built on top of [contenteditable](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content) using [Quill.js](https://quilljs.com/), [Quill's document model parchment](https://github.com/quilljs/parchment) and [React](https://reactjs.org/).
```

Comment:
I don't know if we should really call it a WYSIWYG. That suggests an old school type editor like we have now.